projects/ExtraRepository.cmake

```diff
@@ -0,0 +1,33 @@
+foreach(extra_repository ${EXTRA_REPOSITORIES})
```

Comment:
I have since added a CURA_EXTRA_PROJECTS_DIR variable to the CMake file that is used to include extra CMake files. To be fair, since this is in a separate file there is not much of a problem with keeping it, as long as we also have the extra_projects dir.